### PR TITLE
21142 improve sepolia goerli network icon implementation

### LINKED ISSUE #21142: Improve Sepolia and Goerli network icon display method
### Description
The current approach for displaying the Sepolia and Goerli network icons relies on fetching the background color using the [`getTestNetworkBackgroundColor` selector](https://github.com/MetaMask/metamask-extension/blob/0a215ca2d89958b5bb3902490530b06565114e61/ui/selectors/selectors.js#L650-L660) and then creating the icons using HTML and CSS. This method deviates from how we handle other network icons, adding complexity and increasing the potential for errors and inconsistencies. It would be more advantageous to handle the display of Sepolia and Goerli test networks in the same manner as other built-in networks, using an image and image URL.

### Technical Description
- Incorporate the attached Sepolia and Goerli images provided in this issue.
- Add image links to the Sepolia and Goerli objects within the array returned by `getTestNetworks`.
- Include a deprecation message for `getTestNetworkBackgroundColor` in favor of obtaining the image link.

### Acceptance Criteria
- The network images for Sepolia and Goerli should be accessible from `getTestNetworks` in the same way as other built-in networks, for instance:
  ```javascript
  const testNetworks = getTestNetworks(state);
  <AvatarNetwork src={testNetworks?.rpcPrefs?.imageUrl} />
  ```

### PR DESCRIPTION
**Pull request by jase-b**
## **Description**

**Problem**: The current implementation of Goerli and Sepolia network icons is inconsistent with current practices.

**Solution**: The test networks icons hard-coded in HTML were replaced with the preferred method of rendering icon images dynamically.

1. New image assets added for Sepolia and Goerli icons.
2. Links for new images added to app state data for test networks.
3. Deprecation message added to `getTestNetworkBackgroundColor` selector.

## **Manual testing steps**

1. Open the 'Select a network' menu in MM.
2. View the Goerli and Sepolia icons in the web inspector.
3. Observe the icons are an `` with a `src` referencing their respective, newly added images.
4. Observe the icon letter is a larger size than the previous hard-coded implementation. 

### **Before**

21142_before

### **After**

21142_after

## **Related issues**

Fixes #21142

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've clearly explained:
  - [x] What problem this PR is solving.
  - [x] How this problem was solved.
  - [x] How reviewers can test my changes.
- [x] I’ve indicated what issue this PR is linked to: Fixes #???
- [x] I’ve included tests if applicable.
- [ ] I’ve documented any added code.
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)).
- [x] I’ve properly set the pull request status:
  - [ ] In case it's not yet "ready for review", I've set it to "draft".
  - [x] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Review by georgewrmarshall — COMMENTED** [review]
Hey @jase-b, I know this is a draft but filling out the PR template really helps with review from the team. Would you be able to add link to the issue as well as a detailed description?

File: ui/selectors/selectors.js [hunk 1]
```diff
@@ -647,6 +649,11 @@ export function getShowTestNetworks(state) {
   return Boolean(showTestNetworks);
 }
 
+/**
+ * @deprecated - use getTestNetworks to retrieve image URLs for test
+ * network icons.
```
Comment (jase-b) [inline ui/selectors/selectors.js:654]: Any additions or changes needed in this message, @georgewrmarshall?

Comment (georgewrmarshall) [reply ui/selectors/selectors.js:654]: I think this is fine!

**Review by georgewrmarshall — DISMISSED** [review]
LGTM!

**Review by darkwing — DISMISSED** [review]

**Pull request closed without merging**